src/components/BarChart/BarChart-redux.tsx

```diff
@@ -21,20 +22,35 @@ const BarChartRedux = () => {
 
     const fetchEthereumTransactionsSummed = (transactionsSummedData: TransactionsSummedData): void => {
         dispatch({
-            type: transactionsActions.ETHEREUM_FETCH_TRANSACTIONS_SUMMED,
+            type: transactionsActionsSummed.ETHEREUM_FETCH_TRANSACTIONS_SUMMED,
             transactionsSummedData: transactionsSummedData
         });
     };
+    
+    const fetchEthereumTransactions = (transactionsData: TransactionsData): void => {
+        dispatch({
+            type: transactionsActions.ETHEREUM_FETCH_TRANSACTIONS,
+            transactionsData: transactionsData
+        });
+    };
 
     const fetchEthereumWallets = (): void => {
         dispatch({
           type: walletActions.ETHEREUM_FETCH_WALLETS
         });
       };
 
+    const flushEthereumTransactions = (): void => {
+        dispatch({
+          type: transactionsActions.ETHEREUM_FLUSH_TRANSACTIONS
+        });
+      };
+
```

Comment:
In what case do we need to clear the reducer state of transactions? Just replace the old value with the new one on route param change.